stories/internal/Drilldown.tsx

```diff
@@ -533,6 +533,66 @@ storiesOf('Internal/Drilldown', module)
             </div>
         )
     ))
+    .add('Combo chart with onFiredDrillEvent', () => {
```

Comment:
This screen test is flaky, If you have more than one visualization in one story, you have to define when both visualizations are loaded and ready to take a screen. For that you can use ScreenshotReadyWrapper and you can define via resolver when story content is ready to take screen. See this example: https://github.com/gooddata/gooddata-react-components/blob/38d761e69e96d41898c9684f4396af057fd9ed8a/stories/core_components/AreaChart.tsx#L230

Reply:
IF you will use ScreenshotReadyWrapper please do not forget to define readySelector for ScreenshotReadyWrapper

See:
https://github.com/gooddata/gooddata-react-components/blob/83a6a628047450bf9d9c10d90746148370e506a5/.storybook/test-storybook.config.js#L13

Reply:
In resolver you have access to DOM of wrapper so you can define your logic base on JS etc.

Reply:
Thanks @ivanmjartan, its really helpful for me.
Please help me review again and let me know If I missed something.